lesson01.trivia/index.html

```diff
@@ -1,22 +1,68 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <title>Simple Trivia</title>
+    <title>Shelly's Trivia</title>
     <meta charset="utf-8">
     <meta name="viewport"
           content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <style>
-        body {
 
-        }
-    </style>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+
+    <link rel="stylesheet" type="text/css" href="./index.css">
 </head>
-<body>
+<body class="container">
     <header>
-        <h1>Simple Trivia</h1>
+        <h1 class="my-3 text-info text-center display-4">My Super Trivia - Shelly</h1>
     </header>
     <main>
-        Content Here
+        <h3 class="display-5">Tom yum is a type of hot and sour soup that originated in which country?</h3>
+
+        <form class="text-dark">
+            <div class="custom-control custom-radio">
+                <input class="custom-control-input" name="choice" type="radio" id="Thailand" value="Thailand">
+                <label class="custom-control-label" for="Thailand">Thailand <i class="fas fa-check true Thailand"></i></label>  
```

Comment:
What would you do when you want to change the answer? 
you have to change multiple lines.
Better convention would be to store the correct answer as a variable.